Tests/Resources/app/config/routing.yml

```diff
@@ -0,0 +1,4 @@
+cmf_static_content_index:
```

Comment:
Created for the redirect defintion after delete only. Should be done in a kind of an route enhancer to pass the information by class.